### DISCUSSION TITLE
Github account converted with Enterprise Subscription Converted to Organization

### DISCUSSION THREAD

**Discussion by jaygarg1019** [community/community#142662] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I redeemed a coupon for an Enterprise subscription. I was trying to create an organization but accidentally converted the account to the organization. Now I cannot sign into the account that originally had an Enterprise subscription (it is because of this https://docs.github.com/en/enterprise-cloud@latest/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/converting-a-user-into-an-organization)

I cannot convert organization back to user (https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/converting-an-organization-into-a-user)

What I want help with  @github-staff  - Move my enterprise subscript to the owner account of the organization. 

I also create a ticket "Account deleted Accidently #3067824" but I read this "https://github.com/orgs/community/discussions/43929" so I decided to post this for quick resolution.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @jaygarg1019, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions is through the ticket you opened. 

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
